azure/azure-pr-pipeline.yml

```diff
@@ -35,7 +35,7 @@ extends:
     apigee_deployments:
       - environment: internal-dev
         jinja_templates:
-          GP_CONNECT_ENDPOINTS: gp-connect-prescriptions-management-fhir-endpoints-pr-${{ variables.pr_number }}
+          GP_CONNECT_ENDPOINTS: gp-connect-prescriptions-management-endpoints-pr-${{ variables.pr_number }}
```

Comment:
Why `fhir` has been removed? the project name has `fhir`. This may not cause issue but let's keep it consistent.

Reply:
While implementing KVM, fhir was removed from Key names,hence it was causing issue for finding KVM. Those changes are in infra pipeline under ticket AMB-1519(https://nhsd-jira.digital.nhs.uk/browse/AMB-1519)